src/data/polynomial/eval.lean

```diff
@@ -389,8 +389,8 @@ eval₂_monomial _ _
 @[simp] lemma mul_X_comp : (p * X).comp r = p.comp r * r :=
 begin
   apply polynomial.induction_on' p,
-  { intros p q hp hq, simp [hp, hq, add_mul], },
-  { intros n b, simp [pow_succ', mul_assoc], }
+  { intros p q hp hq, simp only [hp, hq, add_mul, add_comp] },
+  { intros n b, simp only [pow_succ', mul_assoc, monomial_mul_X, monomial_comp] }
```

Comment:
Hey, this is not just moving around stuff, that's cheating! :) But seriously, as long as this didn't happen to any of the moved lines, then it's no big deal.

Reply:
You are right: I had forgotten to mention that I had squeezed some simps.  I will add this to the PR description and check that none of the moved lemmas got squeezed.

If you prefer, I can also split off a further PR which only squeezes the simps.  (My only reason for squeezing them was that it made processing the proofs quicker.)

Reply:
Thank you for updating the description. If you can confirm that none of the moved lemmas got changed, then I see no real need for a further split-off.